definitions/divorce/json/FixedLists/FixedLists.json

```diff
@@ -1265,5 +1265,17 @@
     "ListElementCode": "cheque",
     "ListElement": "Cheque",
     "DisplayOrder": 4
+  },
+  {
+    "LiveFrom": "23/07/2020",
+    "ID": "DocumentType",
+    "ListElementCode": "DispenseWithServiceRefused",
+    "ListElement": "Dispense with service refused"
+  },
+  {
+    "LiveFrom": "23/07/2020",
+    "ID": "DocumentType",
+    "ListElementCode": "DeemedServiceRefused",
+    "ListElement": "Deemed service refused"
```

Comment:
New document types for draft documents.

Reply:
Would this have any side-effect in the CCD UI? If so, then we should probably feature-toggle this.

Reply:
...would confirm. But from the looks of it, if there is no data the documents does not get listed in the tabs section.